Plugins/Alpakit/Source/Alpakit/Private/AlpakitModEntry.cpp

```diff
@@ -1,35 +1,56 @@
 #include "AlpakitModEntry.h"
-
-
 #include "AlpakitSettings.h"
 #include "AlpakitStyle.h"
+#include "IPlatformFilePak.h"
```

Comment:
do you even need that include?

Reply:
Maybe a leftover auto insurance include from rider